Assets/Scripts/Enemies/AIMovement.cs

```diff
@@ -12,16 +12,25 @@ public class AIMovement : MonoBehaviour
     public float currentDirection = 1f; 
     public float platformEdgeDistance = 2;
     public Enemy enemy;
+    public bool randomizeDirection = true;
+    public float randomizeDirectionTime = 1f;
```

Comment:
Suggestion (minor): instead of `public` fields, use `[SerializeField]` instead. `public` should only be used for properties and methods that need to be accessible by other classes.